packages/rainbowkit/src/components/ConnectOptions/DesktopOptions.tsx

```diff
@@ -1,28 +1,38 @@
-import React, { Fragment, useEffect, useState } from 'react';
+import React, { Fragment, useContext, useEffect, useState } from 'react';
 import { touchableStyles } from '../../css/touchableStyles';
 import { isSafari } from '../../utils/browsers';
 import { groupBy } from '../../utils/groupBy';
-import { isMobile } from '../../utils/isMobile';
 import {
   useWalletConnectors,
   WalletConnector,
 } from '../../wallets/useWalletConnectors';
 import { Box } from '../Box/Box';
 import { CloseButton } from '../CloseButton/CloseButton';
 import { ConnectModalIntro } from '../ConnectModal/ConnectModalIntro';
+import { DisclaimerLink } from '../Disclaimer/DisclaimerLink';
+import { DisclaimerText } from '../Disclaimer/DisclaimerText';
 import { BackIcon } from '../Icons/Back';
+import { InfoButton } from '../InfoButton/InfoButton';
 import { ModalSelection } from '../ModalSelection/ModalSelection';
+import { AppContext } from '../RainbowKitProvider/AppContext';
+import { CompactModeContext } from '../RainbowKitProvider/CompactModeContext';
 import { Text } from '../Text/Text';
+
 import {
   ConnectDetail,
   DownloadDetail,
   GetDetail,
   InstructionDetail,
 } from './ConnectDetails';
-import { ScrollClassName, sidebar } from './DesktopOptions.css';
+import {
+  ScrollClassName,
+  sidebar,
+  sidebarCompactMode,
+} from './DesktopOptions.css';
 
 export enum WalletStep {
   None = 'NONE',
+  ProLearn = 'PRO_LEARN',
```

Comment:
Should be `Compact` now.

Also, tiny nit: I feels like it reads better as `LearnCompact` / `LEARN_COMPACT`.

Reply:
agreed. missed this one on the search